manual/embedded-examples/src/docs/dev/procedures.asciidoc

```diff
@@ -46,15 +46,16 @@ These are listed in the table below:
 
 [options="header", cols="m,d"]
```

Comment:
This line is wrong, `cols="m,d"` means there are only two columns in the table, but there are three. A recent commit removed one of the column specifiers, adding it back should fix the table (i.e., making it `cols="m,m,d"`).